web/modules/custom/joinup_user/joinup_user.module

```diff
@@ -342,8 +488,24 @@ function joinup_user_og_membership_presave(EntityInterface $entity) {
 }
 
 /**
- * Implements hook_ENTITY_TYPE_predelete() for OG Membership entities.
+ * Implements hook_form_FORM_ID_alter().
+ *
+ * Disallow access to the cancellation method.
  */
-function joinup_user_og_membership_predelete(EntityInterface $entity) {
-  joinup_user_og_membership_presave($entity);
+function joinup_user_form_user_multiple_cancel_confirm_alter(&$form, FormStateInterface $form_state, $form_id) {
+  joinup_user_form_user_cancel_form_alter($form, $form_state, $form_id);
+}
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ *
+ * Disallow access to the cancellation method.
+ */
+function joinup_user_form_user_cancel_form_alter(&$form, FormStateInterface $form_state, $form_id) {
```

Comment:
Please explain why this is needed. Also, this changes it for all users including UID1. Is it not only needed to restrict access for moderators?

Also as I said yesterday, why do moderators have access to this form in the first place? They have nothing to search here. If it is only for the email messages it would be better to move this to a dashboard entry.

And secondly, this whole exercise is useless since we will make config read-only on production so moderators cannot change ANY of the values in the backend anyway. If they want a change to one of the messages then they will need to create a ticket and this will be updated on the next release.